hexapod_driver/src/hexapod.cpp

```diff
@@ -10,17 +14,26 @@ Hexapod::Hexapod(ros::NodeHandle nh, std::string ns) : _nh(nh), _namespace(ns)
     init();
 }
 
+Hexapod::~Hexapod()
+{
+    ROS_INFO_STREAM("Relaxing...");
+    relax();
+}
+
 void Hexapod::init()
 {
     // Private node handle
     ros::NodeHandle n_p("~");
     // Load Server Parameters
-    n_p.param("Odom", _odom_topic_name, std::string("/odom"));
-    n_p.param("OdomEnable", _odom_enable, true);
-    n_p.param("MoCapOdomTransformEnable", _mocap_odom_enable, false);
+    n_p.param("Odom", _odom_frame, std::string("/odom"));
+    n_p.param("BaseLink", _base_link_frame, std::string("/base_link"));
+    n_p.param("OdomEnable", _odom_enable, false);
+    n_p.param("MoCapOdomEnable", _mocap_odom_enable, true);
```

Comment:
Not really. Check [here](http://wiki.ros.org/Names) for the official conventions.. We can change it to whatever you feel like..